index.html

```diff
@@ -1,134 +1,368 @@
+              <input type="number" class="form-control" id="housenumber" data-i18n="[placeholder]step2.housenumberplaceholder">
+            </div>
+          </div>
+
+          <div class="form-group col-md-3">
+            <label for="city" data-i18n="step2.city"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-cityscape"></i></span>
+              </div>
+              <input type="text" class="form-control" id="city" data-i18n="[placeholder]step2.cityplaceholder">
+            </div>
+          </div>
+          <div class="form-group col-md-2">
+            <label for="postcode" data-i18n="step2.postcode"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-input-numbers"></i></span>
+              </div>
+              <input type="number" class="form-control" id="postcode" data-i18n="[placeholder]step2.postcodeplaceholder">
+            </div>
+          </div>
+        </div>
+
+        <hr>
+        <!-- name and number -->
+        <div class="row mt-3">
+          <div class="form-group col">
+
+            <label for='name' data-i18n="step2.name"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-name-badge"></i></span>
+              </div>
+              <input id='name' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.nameplaceholder" />
+            </div>
+          </div>
+          <div class="form-group col-sm-6">
+            <label for='phone' data-i18n="step2.phone"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-telephone-receiver"></i></span>
+              </div>
+              <input id='phone' type="tel" placeholder='' class="form-control" data-i18n="[placeholder]step2.phoneplaceholder" />
+            </div>
+          </div>
+        </div>
+
+        <!-- website & social -->
+        <div class="row">
+          <div class="form-group col">
+            <label for='website' data-i18n="step2.website"></label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text"><i class="twa twa-desktop-computer"></i></span>
+              </div>
+              <input id='website' type="url" placeholder='' class="form-control" data-i18n="[placeholder]step2.websiteplaceholder" value="https://" />
+            </div>
+          </div>
+          <div class="form-group col-sm-6">
+            <label for='social' data-i18n="step2.social"></label>
+            <input id='social' type='text' placeholder='' class="form-control" data-i18n="[placeholder]step2.socialplaceholder" />
```

Comment:
<i class="twa twa-speech-balloon"></i>

che ne dici?

Reply:
Si, ci avevo pensato proprio alla stessa emoji. Non avevo ancora inserito perché stavo cercando ancora di inserire un' icona che cambiasse a seconda del social, ma alla fine ho lasciato perdere (troppi social e troppo codice per una cosa, in fondo, marginale). Ora la inserisco, grazie 😀